middleware/authz.go

```diff
@@ -0,0 +1,41 @@
+package middleware
+
+// This plugin is based on Casbin: an authorization library that supports ACL, RBAC, ABAC
+// View source at:
+// https://github.com/hsluoyz/casbin
+
+import (
+	"net/http"
+
+	"github.com/hsluoyz/casbin"
```

Comment:
Since this package imports 3rd party pkg, it can't be part of chi/middleware pkg. We could move it to a separate pkg, though. Like `github.com/pressly/chi/middleware/casbin` or to a different repo `github.com/pressly/middleware/casl`. @pkieltyka thoughts?